docs/iot/TOC.md

```diff
@@ -0,0 +1,34 @@
+# [IoT and Tizen IoT](/iot/index.md)
+
+# Getting Started
+## [Tizen IoT](/iot/getting-started/overview.md)
+## [Installing Tizen Studio](/iot/getting-started/tizen-studio-install.md)
+## [Flashing Tizen Images](/iot/getting-started/tizen-image-download-flash.md)
+## [Hardware Configuration](/iot/getting-started/hardware-configuration.md)
+## [Developing Applications with the things sdk api](/iot/getting-started/things-app-development.md)
+## [setting up the smartthings cloud](/iot/getting-started/things-cloud-setup.md)
+## [testing with smartthings app](/iot/getting-started/cloud-app-test.md)
+
+# [Customized Platform Guide](/iot/customized-platform/customized-platform.md)
+
+# API Guides
+## [Overview](/iot/guides/iot-api.md)
+## Things SDK API
+### [Overview](/iot/guides/things-api.md)
+### [Device Definition](/iot/guides/things-api-device.md)
+### [API Usage](/iot/guides/things-api-guide.md)
+## Tizen Peripheral I/O Native API
+### [Overview](/iot/guides/peripheral-io-api.md)
+### [GPIO](/iot/guides/peripheral-io-api-gpio.md)
+### [PWM](/iot/guides/peripheral-io-api-pwm.md)
+### [SPI](/iot/guides/peripheral-io-api-spi.md)
+### [I2C](/iot/guides/peripheral-io-api-i2c.md)
```

Comment:
I meant the string in `[I2C]`, as the TOC list item, not in the url. However as you want to keep it, let's go with it.

Reply:
I already use HTML code in other markdown files. I didn't use it because I didn't know it.
TOC.md is the special file to make a menu tree. When I see the parsing, it can't distinguish HTML tag from Markdown tag.
System engineers know that I2C is I<sub>2</sub>geneally.